.travis.yml

```diff
@@ -1,9 +1,22 @@
-language: vim
+language: ruby
```

Comment:
@cirosantilli I could be mistaken, but I couldn't find any documentation on travis' site regarding `language: vim`. If there is documentation of vim as a travis language, can you point me to it? AFAIK that means that this probably uses their base ruby image currently.

For this PR it doesn't matter what language is at the top so long as I know what versions I can switch on to differentiate stock & latest build. In that respect, the RVM versions are simply well known. I could have just as arbitrarily used python versions. If there is a list of vim versions provided I'll use that.

Regarding bindings, not relevant to my decision process as you can see. If you ever needed specific version of python you can just easily use a `python: 2.6` line in the yml.